portal-impl/src/com/liferay/portal/minifier/MinifierUtil.java

```diff
@@ -104,8 +104,14 @@ private static String _minifyCss(String content) {
 
 			return unsyncStringWriter.toString();
 		}
-		catch (Exception exception) {
-			_log.error("Unable to minify CSS:\n" + content, exception);
+		catch (Throwable throwable) {
+			String failingContent = content;
+
+			if (content.length() > 1073741824) {
```

Comment:
1073741824 is 1GB, right? :thinking: 

1MB = 1024  * 1024  =  1.048.576 

Other than that, LGTM :+1:

Reply:
sorry, my bad.
Changed

Reply:
:+1: